index.html

```diff
@@ -0,0 +1,42 @@
+            alt="beauty hotels" id="header-img">
+        <nav id="nav-bar">
+            <ul class="nav-link">
+                <li>
+                    <a class="nav-link" href="#Home"></a> Home</li>
+                <li>
+                    <a class="nav-link" href="#About us"></a> About Us</li>
+                <li>
+                    <a class="nav-link" href="#contact us"></a> Contact Us</li>
+                <!-- <li>
+                    <a class="nav-link" href="#contact us"></a> Contact Us</li> -->
```

Comment:
Please delete the commented code. This help to make your code clean

Reply:
[OPTIONAL] Its wonderful that you gave the why for your suggestion, I think you could be more suggestive, i.e make suggestions rather than demands 😉